src/iap/shared/pipes/is-control-invalid.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Pipe({
+	name: 'isControlInvalidErrors',
```

Comment:
Typo, I guess. :point_right: :point_left: 
```suggestion
	name: 'isControlInvalid',
```